python/lsst/jointcal/jointcal.py

```diff
@@ -212,8 +216,13 @@ def run(self, dataRefs, profile_jointcal=False):
         associations = jointcalLib.Associations()
 
         load_cat_prof_file = 'jointcal_load_catalog.prof' if profile_jointcal else ''
+        visit_ccd_to_dataRef = {}
+        oldWcsList = []
         with pipeBase.cmdLineTask.profile(load_cat_prof_file):
-            oldWcsList = [self._build_ccdImage(ref, associations, jointcalControl) for ref in dataRefs]
+            for ref in dataRefs:
+                result = self._build_ccdImage(ref, associations, jointcalControl)
+                oldWcsList.append(result.wcs)
+                visit_ccd_to_dataRef[result.key] = ref
```

Comment:
I'm confused? result is a namedtuple, with "wcs" and "key", there's no "Name". I don't know what you're referring to with this comment.